argparse/argparse.py

```diff
@@ -3,7 +3,10 @@
 """
 
 import sys
-from _collections import namedtuple
+try:
+    from _collections import namedtuple
+except ImportError:
+    from collections import namedtuple
```

Comment:
It doesn't need to run under CPython in production but it makes testing a lot easier by comparing the output of running the same test under CPython and uPy. We could have just
```
from collections import namedtuple
```
if this module depended on collections.